src/libraries/Common/tests/ComplianceTests/GB18030/Tests/DirectoryInfoTests.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+
+namespace GB18030.Tests;
+
+public class DirectoryInfoTests : DirectoryTestBase
+{
+    protected override void CreateDirectory(string path) => new DirectoryInfo(path).Create();
+    protected override void DeleteDirectory(string path, bool recursive) => new DirectoryInfo(path).Delete(recursive);
+    protected override void MoveDirectory(string source, string destination) => new DirectoryInfo(source).MoveTo(destination);
+
+    [Theory]
+    [MemberData(nameof(TestHelper.DecodedTestData), MemberType = typeof(TestHelper))]
+    public void CreateSubdirectory(string decoded)
+    {
```

Comment:
do we need to do clean up after executing every test in this file?

Reply:
This apply on other directory and file tests

Reply:
I am not seeing the dispose is explicitly called here. Do you mean the temp directory has a finalizer that eventually likely calling the dispose? should we do teh following instead?

using var subDirInfo = TempDirectory.CreateSubdirectory(gb18030Line);

Reply:
The dispose is in the base class: https://github.com/dotnet/runtime/blob/ba384e52b752ff8632dbf8727ed34b182fc82b57/src/libraries/Common/tests/ComplianceTests/GB18030/Tests/DirectoryTestBase.cs#L77